ui/app/common/displaycontrols/dashboard/directives/dashboard.js

```diff
@@ -1,12 +1,49 @@
 'use strict';
 
 angular.module('bahmni.common.displaycontrol.dashboard')
-
-    .directive('dashboard', [function () {
-        var controller = function ($scope, $filter) {
+    .directive('dashboard', ['appService', '$stateParams', '$bahmniCookieStore', 'configurations', 'encounterService', 'spinner', 'auditLogService', 'messagingService', '$state', '$translate', function (appService, $stateParams, $bahmniCookieStore, configurations, encounterService, spinner, auditLogService, messagingService, $state, $translate) {
+        var controller = function ($scope, $filter, $rootScope) {
             var init = function () {
                 $scope.dashboard = Bahmni.Common.DisplayControl.Dashboard.create($scope.config || {}, $filter);
             };
+            $scope.tabConfigName = $stateParams.tabConfigName || 'default';
+
+            if ($scope.patient !== undefined) {
+                $scope.formData = {
+                    patientUuid: $scope.patient.uuid,
+                    patient: $scope.patient,
+                    encounterUuid: $scope.activeEncounterUuid,
+                    showEditForActiveEncounter: $scope.config.sections['forms-v2-react'] && $scope.config.sections['forms-v2-react'].dashboardConfig && $scope.config.sections['forms-v2-react'].dashboardConfig.showEditForActiveEncounter || false,
```

Comment:
Priority: MINOR

The config name "showEditForActiveEncounter" seems to be misleading when it is set to false. When set to false, the edit icon is shown for all including the active encounter. So, better to rename  to "showEditOnlyForActiveEncounter"